scripts/verdaccio.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+port=4000
+original_registry=`npm get registry`
+registry="http://localhost:$port"
+output="output.out"
+ci=false
+commit_to_revert="HEAD"
+
+if [ "$1" = "ci" ];
+then
+  ci=true
+fi
+
+function cleanup {
+  echo "Cleaning up"
+  if [ "$ci" = false ];
+  then
+    lsof -ti tcp:4000 | xargs kill
+    # Clean up generated dists if run locally
+    rm -rf packages/**/dist
+    rm -rf storage/ ~/.config/verdaccio/storage/ $output
+    git tag -d $(git tag -l)
+    git fetch
+    git reset --hard $commit_to_revert
+    npm set registry $original_registry
+    return
+  else
+    # lsof doesn't work in circleci
+    netstat -tpln | awk -F'[[:space:]/:]+' '$5 == 4000 {print $(NF-2)}' | xargs kill
+    return
+  fi
+}
+
+trap cleanup ERR EXIT
+
+# Generate dists for the packages
+yarn parcel build packages/@adobe/react-spectrum/ packages/@react-{spectrum,aria,stately}/*/ packages/@internationalized/*/ --no-optimize --log-level error
```

Comment:
Maybe use `make build` here so we avoid duplication?

Also `@adobe/react-spectrum` shouldn't be built by Parcel.

Reply:
Hrm, if we don't build `@adobe/react-spectrum` how do I get it to publish to verdaccio for consumption in the test app?

Reply:
The mono packages have a `prepublishOnly` script that runs as part of publishing.

Reply:
Hm, interesting, those prepublish scripts should be running via this script as well, lemme try it again

Reply:
It shouldn't be any different than our usual nightly or release publishing steps I think

Reply:
yep, confirmed that `make build` works locally, not sure why I saw otherwise when helping with the test app PR...